public/app/features/alerting/unified/hooks/useAbilities.test.tsx

```diff
@@ -137,9 +144,20 @@ describe('alertmanager abilities', () => {
 });
 
 describe('AlertRule abilities', () => {
+  const server = setupMswServer();
   it('should report that all actions are supported for a Grafana Managed alert rule', async () => {
     const rule = getGrafanaRule();
 
+    // TODO: Remove server mocking within test once server is run before all tests
+    mockFolderApi(server).folder(
+      (rule.rulerRule as RulerGrafanaRuleDTO).grafana_alert.namespace_uid,
+      mockFolder({
+        accessControl: { [AccessControlAction.AlertingRuleUpdate]: false },
+      })
+    );
+    mockAlertmanagerChoiceResponse(server, defaultAlertmanagerChoiceResponse);
+    server.listen();
```

Comment:
Why do we need to call `listen()` in here? `setupMswServer` is calling listen in the `beforeAll`

Reply:
Good spot! Thanks, forgot this one. I've removed it :+1: